channeldb/db.go

```diff
@@ -730,6 +731,194 @@ func (c *ChannelStateDB) FetchChannelByID(tx kvdb.RTx, id lnwire.ChannelID) (
 	return c.channelScanner(tx, selector)
 }
 
+// ChanCount is used by the server in determining access control.
+type ChanCount struct {
+	OpenClosed bool
```

Comment:
nit: consider renaming to `HasOpenOrClosedChan` for better readability. Also please add godocs to these exported fields.